src/games/strategy/util/LocalizeHTML.java

```diff
@@ -142,33 +142,4 @@ public static String localizeImgLinksInHTML(final String htmlText, final Resourc
 
     return rVal;
   }
-
```

Comment:
Converting to a straight unit test is not a good idea, and we are missing information - IE: the expected counts.

The overall quality of the test would be poor, there is a lot of redundant and overly complex test data. Most importantly there are many test cases missing: empty data, empty string, blank space string, just the html open close, image tag by itself, the various permutations of src values within an image tag.  Similar for the other methods that do parsing.

With that, this is hardly the starting point. This is perhaps the last test to be written, and it would use all of the scaffolding we had built earlier. Decomposing this into those pieces is going to take more time to than to just write the test.

Then the question is, why are we adding test coverage here, and not elsewhere?
So while this does give what the overall pattern of a unit test would look like, the test data in this case is not very valuable, and the remaining stuff is relatively trivial / obvious if we were to test it.